src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -107,15 +107,13 @@ class CrashModule(
                     javaVersion = it.replace("Java Version: ", "", true)
 
                 if (it.contains("Is Modded", true))
-                    modded = !it.contains("Probably Not", true)
+                    modded = !it.contains("Probably Not", true) && !it.contains("Unknown", true)
             }
 
             if (minecraftVersion != null && javaVersion != null && lines.size >= 7)
                 CrashInfo(
                     CrashInfoType.MINECRAFT,
                     exception?.trim(),
-                    minecraftVersion!!.trim(),
-                    javaVersion!!.trim(),
```

Comment:
Shouldn't the collection of these values also be removed for #35?
In here, you only removed minecraft and java version from CrashInfo, but they are still collected in local variables in here:
https://github.com/mojira/arisa-kt/blob/067d3e422afcedda1bdd0fef617d54d80d07ae5a/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt#L96-L111
and here the java version:
https://github.com/mojira/arisa-kt/blob/067d3e422afcedda1bdd0fef617d54d80d07ae5a/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt#L123-L133

Reply:
First example is used in line 113

Reply:
second one is used in line 135

Reply:
There is no reason to check for null, when not needing the value, is there?
The null check isn't even there in arisa-js IIRC

Reply:
If I understand the logic correctly, it doesnt actually return it as a valid crash if it doesnt have that.

In Arisa I remembered that we checked if it was a minecraft or java crash that way

Reply:
The check whether it's a minecraft or java crash is here:
https://github.com/mojira/arisa-kt/blob/c1242e91c43c7f019ec33ebde49ecac8bf9a300c/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt#L95
and here:
https://github.com/mojira/arisa-kt/blob/c1242e91c43c7f019ec33ebde49ecac8bf9a300c/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt#L122
and checks if one of those constants are contained in the file:
https://github.com/mojira/arisa-kt/blob/c1242e91c43c7f019ec33ebde49ecac8bf9a300c/src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt#L17-L18

The null check was just a sanity check I added which is not present in the original module.

Reply:
done